nltk/embedding/prune_word2vec_model.py

```diff
@@ -0,0 +1,60 @@
+# Natural Language Toolkit: Word2Vec Wrapper through Gensim
+#
+# Copyright (C) 2001-2015 NLTK Project
+# Author: Long Duong <longdt219@gmail.com>
+# URL: <http://nltk.org/>
+# For license information, see LICENSE.TXT
+"""
+Supporting code to extract part of the pre-trained model (GoogleNews-vectors-negative300.bin.gz) 
+from  https://code.google.com/p/word2vec/
+ """
+ 
+from __future__ import absolute_import, division, unicode_literals
+import gensim
+from gensim.models.word2vec import Word2Vec
+
+if __name__ == '__main__':
+    # Load the C pre-trained model file 
+    model = Word2Vec.load_word2vec_format('GoogleNews-vectors-negative300.bin.gz', binary = True);
+    
+    # Only output word that appear in the Brown corpus 
+    from nltk.corpus import brown
+    dict = {}  
+    for word in brown.words():
+        dict[word] = 1 
+    print (len(dict.keys()))
+    
+    # Only print out the model that contain word in dict
+    out_file = 'pruned.word2vec.txt'
+    f = open(out_file,'wb')
```

Comment:
Please consider using a context manager to open a file, so you don't need to close it manually.

```` python
with open(out_file, 'wb') as f:
    ...
````